### DISCUSSION TITLE
Mini-event changer de direction

### DISCUSSION THREAD

**Discussion by Gabrielre14** [Crownicles/Crownicles#910] [Nouvelles idées]
Tout le concept est dans le titre
L'idée serait que, grâce à un mini-event qui te déboussole, tu ne saches plus dans quelle direction tu allais, et que tu doives choisir une nouvelle direction...
De cette manière, en plein milieu d'un voyage, un voyageur pourra rebrousser chemin, aller à un autre endroit, ou continuer sa route...
Bien sûr, les possibilités sont données de sorte que le voyage total ne puisse pas durer plus de 10h !
Une petite idée qui m'a soudainement traversé l'esprit en voyant quelqu'un avec peu de vie se diriger vers la vallées de rois 😉

**Comment by BastLast** [comment]
Non ça peut être marrant en vrai si le temps question est extrêmement rare histoire que les gens ne puisse pas compter dessus
Aussi j'aimerais qu'il y a un coût au fait de faire demi-tour parce que ça peut très vite être pété dans certaines zones aussi

**Comment by Gabrielre14** [comment]
L'idée serait plutôt qu'il ait le choix entre plusieurs endroits, comme à la fin d'un évènement.
Bien sûr, il faut encore développer la rédaction et la mise en page, par exemple : comment on peut écrire ça sur les !r suivants

**Reply by BastLast** [reply]
Si tu es au milieu d'un chemin, on peut pas te teleporter ailleurs...

**Comment by Gabrielre14** [comment]
Non c'est pas vraiment ça
tout simplement tu es à Boug coton, en direction de la vallée des rois
Au bout de 3 heures tu reçois le mini-event : 
soit tu vas à boug coton (il te reste 3h de voyage)
soit tu vas à la vallée des rois (il te reste 4h je crois)
soit tu vas au croisement des destins (il te reste 5h par exemple)

la destination finale est modifiée, le temps qu'il reste aussi

et des fois on aura plus de choix que d'autres (2 ou 3, comme le choix des lieux après les évènements)

**Reply by BastLast** [reply]
Ok du coup ca c'est vraiment compliqué à faire :/ du coup a priori ca sera nope
